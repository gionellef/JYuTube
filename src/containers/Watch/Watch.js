import React from 'react';

import { Video } from '../../components/Video/Video';
import { VideoPreview } from '../../components/VideoPreview/VideoPreview';

import './Watch.scss';

export class Watch extends React.Component {
  render() {
    return (
      <div className='watch-component'>
        <VideoPreview />
        <VideoPreview horizontal={false} />
        <VideoPreview horizontal={true} />
        
      </div>
      // <div style={{maxWidth: '80%'}}>
      //   <Video id='P5ktQh3sWZk' />
      // </div>
    );
  }
}