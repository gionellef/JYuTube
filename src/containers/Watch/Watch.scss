@import '../../styles/shared.scss';

.watch-component {
  margin-top: $header-nav-height;
}