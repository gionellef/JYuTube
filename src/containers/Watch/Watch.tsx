import * as React from 'react';

import Video from '../../components/Video/Video';
import RelatedVideos from '../../components/RelatedVideos/RelatedVideos';
import VideoMetadata from '../../components/VideoMetadata/VideoMetadata';
import { VideoInfoBox } from '../../components/VideoInfoBox/VideoInfoBox';

import './Watch.scss';
import { Comments } from '../Comments/Comments';

export class Watch extends React.Component {
  render() {
    return (
      <div className='watch-grid'>
        <Video className='video' id='P5ktQh3sWZk' />
        <VideoMetadata className='metadata' viewCount={1000} />
        <VideoInfoBox className='video-info-box' />
        <Comments className='comments' numberOfComments={52374} />
        <RelatedVideos className='related-videos' />
      </div>

    );
  }
}