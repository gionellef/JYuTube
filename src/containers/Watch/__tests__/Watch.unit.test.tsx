import * as React from 'react';

import { shallow } from 'enzyme';

import { Watch } from '../Watch';

describe('Watch', () => {
  test('renders Watch view', () => {
    const wrapper = shallow(
      <Watch />
    );

    expect(wrapper).toMatchSnapshot();
  });
});