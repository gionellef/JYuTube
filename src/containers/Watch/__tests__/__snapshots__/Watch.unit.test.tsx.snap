// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Watch renders Watch view 1`] = `
<div
  className="watch-grid"
>
  <Video
    className="video"
    id="P5ktQh3sWZk"
  />
  <VideoMetadata
    className="metadata"
    viewCount={1000}
  />
  <VideoInfoBox
    className="video-info-box"
  />
  <Comments
    className="comments"
    numberOfComments={52374}
  />
  <RelatedVideos
    className="related-videos"
  />
</div>
`;
